components/Card.js

```diff
@@ -1,8 +1,9 @@
 import styled from "styled-components";
 
-export default function Card({ date, weight, height }) {
+export default function Card({ date, weight, height, handleDelete, id }) {
   return (
     <CardElement>
+      <DeleteButton onClick={() => handleDelete(id)}>X</DeleteButton>
```

Comment:
The button should have an accessable text (e.g. aria-label)

Reply:
this is very true!

Reply:
gave it an aria-label and a button type :)